### ISSUE TITLE
Protocol methodologies

### ISSUE THREAD

**Issue by ishraq8** [messari/subgraphs#1059]
This story is for 24-27 in the spreadsheet:

https://docs.google.com/spreadsheets/d/1_-tOe3ebx3ZM29L4h792A_Z62ztEJ6usmPZJLvENe7g/edit#gid=716614101

Messari - 16. Maple Finance

Messari - 17. TrueFi

Messari - 18. Alpha Finance

Messari - 24. BiSwap

Messari - 25. cBridge

Messari - 26. Synapse

Messari - 27. Arrakis Finance

**Comment by this-username-is-taken** [comment]
I'd imagine for Maple/TrueFi/Arrakis it'll just be consolidating with our existing methodology?

Alpha Finance (specifically Homora) should use the generic schema.

cBridge and Synapse should be used to come up with the new bridges schema

**Comment by ishraq8** [comment]
Yes, some of the consolidated schemas are already done and were updated before. Will work on Alpha and do the bridges at the end because they are new schemas.

**Comment by mehmetbybz** [comment]
https://docs.google.com/spreadsheets/d/1_-tOe3ebx3ZM29L4h792A_Z62ztEJ6usmPZJLvENe7g/edit#gid=716614101

**Comment by ishraq8** [comment]
Pull request submitted: 

https://github.com/messari/subgraphs/pull/1217
